README.md

```diff
@@ -42,6 +42,10 @@ A dedicated `etcd` cluster for each tenant cluster doesn’t scale well for a ma
 
 With this solution, the resiliency is guaranteed by the usual `etcd` mechanism, and the pods' count remains under control, so it solves the main goal of resiliency and costs optimization. The trade-off here is that we have to operate an external `etcd` cluster and manage the access to be sure that each tenant cluster uses only its data. Also, there are limits in size in `etcd`, defaulted to 2GB and configurable to a maximum of 8GB. We’re solving this issue by pooling multiple `etcd` and sharding the tenant control planes.
 
+## Getting started
+
+Please refer to the [getting started documentation](./docs/getting-started-with-kamaji.md) to deploy a minimal setup of Kamaji.
```

Comment:
I would suggest 

Please refer to the [Getting Started guide](./docs/getting-started-with-kamaji.md) to deploy a minimal setup of Kamaji on KinD

Reply:
Applied